### ISSUE TITLE
Index out of range

### ISSUE THREAD

**Issue by mazzy89** [tahvane1/jablotron80#46]
Another error I have found coming from this integration is

```
Unexpected error: Traceback (most recent call last): File "/config/custom_components/jablotron80/jablotron.py", line 709, in read_send_packet_loop records_tmp = self._read_data() File "/config/custom_components/jablotron80/jablotron.py", line 661, in _read_data if data[1 + int(size)] == 0xff: IndexError: index out of range
```

Need to find out how to reproduce it but maybe I get it when I arm the alarm through the home assistant integration.

**Comment by mattsaxon** [comment]
I have experienced this myself now and have an idea what the issue is

**Comment by mattsaxon** [comment]
Fixed in release 0.27
